helpers/metadata.go

```diff
@@ -1,35 +1,96 @@
 package helpers
 
-import "sigs.k8s.io/controller-runtime/pkg/client"
+import (
+	"testing"
+
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+var (
+	testObject      = &tektonv1beta1.PipelineRun{}
+	testAnnotations = map[string]string{
+		"a.test/annotation":              "foo",
+		"another.test/annotation-string": "bar",
+	}
+	testLabels = map[string]string{
+		"a.test/label":              "foo",
+		"another.test/label-string": "bar",
+	}
+)
+
+func init() {
+	testObject.SetAnnotations(testAnnotations)
+	testObject.SetLabels(testLabels)
+}
 
 // HasAnnotation checks if a given annotation exists
-func HasAnnotation(object client.Object, annotation string) bool {
-	_, found := object.GetAnnotations()[annotation]
+func HasAnnotation(t *testing.T) {
+	var tests = []struct {
+		annotation string
+		want       bool
+	}{
+		{"a.test/annotation", true},
+		{"bar", false},
+	}
 
-	return found
+	for _, tt := range tests {
```

Comment:
What does **tt** stands for?

Reply:
For table driven tests, this variable is called `tt` instead of `test`.